sdk/personalizer/azure-ai-personalizer/samples/multi_slot_rank_actions_and_reward_events.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: multi_slot_rank_actions_and_reward_events.py
+DESCRIPTION:
+    This sample demos sending a rank and reward call to personalizer for multi-slot configuration.
+USAGE: python multi_slot_rank_actions_and_reward_events.py
+"""
+from util import get_personalizer_client
+
+
+def main():
+    client = get_personalizer_client()
```

Comment:
In general we try to structure samples so that the entire scenario is copy/pasteable in one file.

Reply:
Will do. This will result in the code from util.py to be duplicated.